backend/src/controllers/review.controller.ts

```diff
@@ -29,6 +41,49 @@ class ReviewController extends BaseController<IReview> {
     return res.status(httpStatus.OK).send(analysis);
   }
 
+  async generateResponseForReview(req: AuthRequest<{}, IReviewReply, IGenerateReviewReply>, res: Response) {
+    // TODO: implement this function with calls to OpenAPI
```

Comment:
Is there a task for it? relevant for every TODO here

Reply:
Yes.
Reply generation is #70
Get reviews is #67

Reply:
Makik